Realm/RLMRealm.h

```diff
@@ -401,11 +401,11 @@ typedef void (^RLMNotificationBlock)(RLMNotification notification, RLMRealm *rea
 
  @warning This method may only be called during a write transaction.
 
- @param array  An `NSArray`, `RLMArray`, or `RLMResults` of `RLMObject`s (or subclasses) to be added to the Realm.
+ @param array  An enumerable object such as `NSArray` or `RLMResults` which contains objects to be added to the Realm.
 
  @see   `addOrUpdateObject:`
  */
-- (void)addOrUpdateObjectsFromArray:(id)array;
+- (void)addOrUpdateObjects:(NSObject<NSFastEnumeration> *)array;
```

Comment:
Renaming `addOrUpdateObjectsFromArray` to `addOrUpdateObjects` is a source-breaking change. Should we also retain (and deprecate) the old name?

(Note that the method was renamed for consistency with `addObjects`.)